package.json

```diff
@@ -12,20 +12,22 @@
     "Maël Thomas-Quillévéré <mael.thomas@data.gouv.fr>"
   ],
   "description": "Un simulateur de coût d'embauche",
-  "engines": { "node": ">=0.12" },
+  "engines": {
+    "node": ">=0.12"
+  },
   "dependencies": {},
   "devDependencies": {
     "autoprefixer": "^6.0.3",
-    "css-loader": "^0.18.0",
+    "css-loader": "^0.22.0",
```

Comment:
According to [this](https://docs.npmjs.com/misc/semver#caret-ranges-1-2-3-0-2-5-0-0-4), carret and tilde are equivalent for 0.22.0 (because of the zero).

> allows patch and minor updates for versions 1.0.0 and above, patch updates for versions 0.X >=0.1.0, and no updates for versions 0.0.X.